src/app/components/client/Modal.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../tokens";
+
+.modalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5); /* Semi-transparent black overlay */
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .modal {
+    background-color: $color-grey-05;
+    padding: $spacing-lg;
+    border-radius: $border-radius-md;
+    max-width: $content-md;
+    position: relative;
+
+    .modalContent {
+      padding-top: $spacing-lg;
+
+
+    .closeButton {
+      position: absolute;
+      top: 0;
+      right: 0;
+      padding: $spacing-md;
+      color: $color-grey-40;
+      background: none;
+      border: none;
+      cursor: pointer;
+
+      &:hover {
+        color: $color-grey-30;
+      }
+    }
```

Comment:
Formatting: The block should `.closeButton` have an additional level of indentation.